tests/large/test_ide.py

```diff
@@ -40,7 +40,9 @@ class EclipseIDETests(LargeFrameworkTests):
     def setUp(self):
         super().setUp()
         self.installed_path = os.path.join(self.install_base_path, "ide", "eclipse")
-        self.desktop_filename = "eclipse.desktop"
+        self.desktop_filename = "eclipse-java.desktop"
+        self.command_args = '{} ide eclipse'.format(UMAKE)
```

Comment:
I see that in medium tests you are calling a similar variable umake_command_args, so maybe just define it here and remove the override in medium tests (as it's specifying the same thing?)